index.js

```diff
@@ -980,6 +983,11 @@ function shimPromiseAPI (RTCPeerConnection, pc) {
   }
 }
 
+// HACK: Filter trickle lines when trickle is disabled #354
+function filterTrickle (sdp) {
+  return sdp.replace(new RegExp('a=ice-options:trickle\\s\\n', 'g'), '')
```

Comment:
Why not `/a=ice-options:trickle\s\n/g`? Seems to be easier to read.

Reply:
Sure. I forgot that syntactic sugar with regexes. 👍